docs/features/rate-limiter.md

```diff
@@ -40,7 +40,7 @@ export class BackendService {
 Note: The rate limiter will work only on asynchronous functions.  
 :::
 :::info
-Important: The rate limiter decorator **must** be used on a method that has the first parameter as `GnzContext` and the rest of the parameters are the ones you want to pass to the method. Even if you won't explicitly use the `GnzContext` parameter, it must be there. This is because the context needs to be populated with the IP address of the request. This will be done automatically by the rate limiter decorator. To learn more about the `GnzContext` object, see the [documentation](https://genezio.com/docs/features/backend-deployment/)
+Important: The rate limiter decorator **must** be used on a method that has the first parameter as `GnzContext` and the rest of the parameters are the ones you want to pass to the method. Even if you won't explicitly use the `GnzContext` parameter, it must be there. This is because the context needs to be populated with the IP address of the request. This will be done automatically by the rate limiter decorator. To learn more about the `GnzContext` object, see the [documentation](/docs/project-structure/genezio-context)
```

Comment:
`and the rest of the parameters are the ones you want to pass to the method`
this phrase doesn't ring quite well. maybe we should rephrase it. try asking a LLM to do that :))